administrator/components/com_media/libraries/media/file/adapter/local.php

```diff
@@ -280,10 +280,16 @@ private function getPathInformation($path)
 
 		try
 		{
-			// Get the image properties
-			$props       = JImage::getImageFileProperties($path);
-			$obj->width  = $props->width;
-			$obj->height = $props->height;
+			$obj->width  = 0;
+			$obj->height = 0;
+
+			if (!is_dir($path))
+			{
+				// Get the image properties
+				$props       = JImage::getImageFileProperties($path);
+				$obj->width  = $props->width;
+				$obj->height = $props->height;
+			}
```

Comment:
Most information you find on the net do say that getimagesize is the function to check if it is an image or not. That is what the JImage lib is doing. I'm not sure if it would be enough to rely on the filetyope or mime type only.